server/src/main/java/org/jboss/as/server/parsing/StandaloneXml.java

```diff
@@ -528,16 +527,16 @@ private void parseHttpManagementInterface1_0(final XMLExtendedStreamReader reade
                 final Attribute attribute = Attribute.forName(reader.getAttributeLocalName(i));
                 switch (attribute) {
                     case INTERFACE: {
-                        HttpManagementResourceDefinition.INTERFACE.parseAndSetParameter(value, mgmtSocket, reader);
+                        //ignore
                         hasInterfaceName = true;
```

Comment:
That is the same debate we had with Darren on the subject, but agreed that no solution is perfect. 
If you want to go with failing instead of ignoring, i will fix it so.